akka-cluster-bootstrap/src/main/scala/com/lightbend/rp/akkaclusterbootstrap/ClusterBootstrap.scala

```diff
@@ -20,12 +20,17 @@ import akka.actor._
 import akka.cluster.Cluster
 import akka.cluster.bootstrap.{ ClusterBootstrap => Bootstrap }
 import akka.cluster.http.management.ClusterHttpManagement
+import com.lightbend.rp.common.Platform
 
 final class ClusterBootstrap(system: ExtendedActorSystem) extends Extension {
   val cluster = Cluster(system)
 
   if (cluster.settings.SeedNodes.nonEmpty) {
     system.log.warning("ClusterBootstrap is enabled but seed nodes are defined, no action will be taken")
+  } else if (Platform.active == None) {
+    // When running locally, for example with "sbt runAll", actors join local cluster.
```

Comment:
I think Lagom is supposed to be doing this for us. I'll have to do some more digging, I think for local we don't want to do anything.

Reply:
Removed cluster self-join here, thanks. I checked and it indeed works now with newest sbt-reactive-app.